src/components/implementation/no_interface/llbooter/boot_deps.h

```diff
@@ -113,6 +122,63 @@ boot_spd_initaep_get(spdid_t spdid)
 	return cos_sched_aep_get(boot_spd_defcompinfo_get(spdid));
 }
 
+/* The fault handlers*/
+void
+fh_div_by_zero_err(unsigned long sp, unsigned long ip, unsigned long fault_addr, unsigned long fault_type)
+{
+	PRINTLOG(PRINT_DEBUG, "in div by zero error fault handler, fault happens in component:%u\n\n", cos_inv_token());
+	call_cap_op(0, 0, 0, 0, 0, 0);
+	return;
+}
+
+void
+fh_memory_access(unsigned long sp, unsigned long ip, unsigned long fault_addr, unsigned long fault_type)
+{
+	PRINTLOG(PRINT_DEBUG, "in memory access handler, memory access fault happens in component:%u\n\n", cos_inv_token());
```

Comment:
We likely want to print out all of the registers as well.  I'm guessing (but haven't looked further in the code yet) that you don't yet have mechanisms for accessing the thread's registers.